src/Components/Fsi.fs

```diff
@@ -494,13 +496,27 @@ module Fsi =
             return terminal
         }
 
+    let private commentRegex =
+        Regex(@"\/{2,3}.+\S+", RegexOptions.ECMAScript)
```

Comment:
This won't match the following:
```
//a
<any number of spaces here>//a<any number of spaces here>
```
First is quite short, so I suppose it's not an issue, second one might cause some issues?

This will match the following:
```
let foo = "// asdasdasdasdasdasd"
```

Which probably is not desired.

Second regex probably has some similar issues.